package.json

```diff
@@ -17,7 +17,7 @@
     "develop": "gatsby develop --host localhost --port 8000",
     "format": "prettier --trailing-comma es5 --no-semi --single-quote --write 'src/**/*.js'",
     "test": "npm run lint",
-    "test:e2e": "E2E_OUTPUT=${E2E_OUTPUT-./e2e/test_results} E2E_ERRORSHOTS_OUTPUT=${E2E_ERRORSHOTS_OUTPUT-./e2e/errorShots} E2E_SCREENSHOTS=${E2E_SCREENSHOTS-./e2e/screenshots/} wdio",
+    "test:e2e": "E2E_OUTPUT=${E2E_OUTPUT-./e2e/test_results} E2E_ERRORSHOTS_OUTPUT=${E2E_ERRORSHOTS_OUTPUT-./e2e/errorShots} E2E_SCREENSHOTS=${E2E_SCREENSHOTS-./e2e/screenshots/} E2EPROFILE=${E2EPROFILE-sauceextended} wdio",
```

Comment:
this default will be the default on local computers so this should just be, chrome, right?

Reply:
If you write nothing on a local computer (where CI != truthy or TEST_PROVIDER != sauce) this runs headless chrome. If you use e.g. E2EPROFILE=firefox the tests will run on Firefox.

Reply:
didn't we stop using `TEST_PROVIDER`?

Reply:
didn't we agree that we remove the `CI` check?